plugins/modules/vios.py

```diff
@@ -125,6 +139,26 @@
             - Default value is False.
             - Valid only for C(state) = I(facts)
         type: bool
+    image_dir:
+        description:
+            - Specifies the directory on the remote server for the VIOS installation image.
```

Comment:
- This parameter determines whether to use the HMC based image for VIOS installation, rather than relying on the Network Installation Manager (NIM).
- Name of the directory on which VIOS image is available on HMC. 
- This represent the same `directory_name` that is used during the `copy` action